tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp

```diff
@@ -685,6 +769,22 @@ SPECTRE_TEST_CASE("Unit.ApparentHorizons.StrahlkorperGr.AreaElement",
             expected_area, expected_irreducible_mass);
 
   test_euclidean_area_element(kerr_horizon);
+
+  test_integral_correspondence(gr::Solutions::Minkowski<3>{},
+                               kerr_horizon);
+  test_integral_correspondence(gr::Solutions::KerrSchild{mass, spin, center},
+                               kerr_horizon);
+
+  // Check that this still works for a surface inside and outside the horizon
```

Comment:
"this still works" -> "the two expressions giving the surface integral are still equal" or some rephrasing that increases clarity.